docs/source/enable_cc_lifecycle.md

```diff
@@ -0,0 +1,300 @@
+endorsement policies for the new chaincode lifecycle must be added to each
+channel.
+
+You can set the same environment you set when updating the peer organizations.
+Note that in this case you will not be updating the configuration of an org in
+the configuration, so the `ORGNAME` variable will not be used.
+
+Once you have set the environment variables, navigate to
+[Step 1: Pull and translate the config](./config_update.html#Step-:-Pull-and-translate-the-config).
+
+Once you have a `modified_config.json`, add the channel endorsement policy
+(as listed in `enable_lifecycle.json`) using this command:
+
+```
+jq -s '.[0] * {"channel_group":{"groups":{"Application": {"policies": .[1].appPolicies}}}}' config.json ./enable_lifecycle.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#Step-3-Re-encode-and-submit-the-config).
+
+For this channel update to be approved, the policy for modifying the `Channel/Application`
+section of the configuration must be satisfied. By default, this is a `MAJORITY`
+of the peer organizations on the channel.
+
+## Edit channel ACLs (optional)
+
+You can choose to overwrite the default [Access Control List (ACL)](./access_control.html)
+for these new channel endorsement policies by performing a channel update.
+
+If you look at `enable_lifecycle.json`, we can see the fields being added:
+
+```
+"acls": {
+ "_lifecycle/CommitChaincodeDefinition": {
+   "policy_ref": "/Channel/Application/Writers"
+ },
+ "_lifecycle/QueryChaincodeDefinition": {
+   "policy_ref": "/Channel/Application/Readers"
+ },
+ "_lifecycle/QueryNamespaceDefinitions": {
+   "policy_ref": "/Channel/Application/Readers"
+```
+
+You can leave the same environment in place as when you previously edited
+application channels.
+
+Once you have the environment variables set, navigate to
+[Step 1: Pull and translate the config](./config_update.html#Step-:-Pull-and-translate-the-config).
+
+Once you have a `modified_config.json`, add the ACLs (as listed in
+`enable_lifecycle.json`) using this command:
+
+```
+jq -s '.[0] * {"channel_group":{"groups":{"Application": {"values": {"ACLs": {"value": {"acls": .[1].acls}}}}}}}' config.json ./scripts/policies.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#Step-3-Re-encode-and-submit-the-config).
+
+For this channel update to be approved, the policy for modifying the `Channel/Application`
+section of the configuration must be satisfied. By default, this is a `MAJORITY`
+of the peer organizations on the channel.
+
+## Enable new lifecycle in `core.yaml`
+
+If you follow [the recommended process](./upgrading_your_components.html#Overview)
+for using a tool like `diff` to compare the new version of `core.yaml` packaged
+with the binaries with your old one, you will not need to whitelist
+`_lifecycle: enable*` under `chaincode/system`.
```

Comment:
There is an extra `*` in `_lifecycle: enable*` - should be `_lifecycle: enable`. Also suggest to make it clear that the new core.yaml has added `_lifecycle: enable` to `chaincode/system` - e.g., revise the last sentence to:
you will not need to whitelist `_lifecycle: enable*` because the new core.yaml has added it under `chaincode/system`.

Reply:
Should be done now.